### ISSUE TITLE
[Feature Request] Ability to specify internal frontend in Helm chart

### ISSUE THREAD

**Issue by Meeki1** [temporalio/helm-charts#560]
Since version 1.20 it became possible to use internal frontend. It would be nice if it was possible to use it in helm chart too

**Comment by mxk** [comment]
It looks like support for internal-frontend has been added. However, the web UI still uses [frontend](https://github.com/temporalio/helm-charts/blob/c0eaf12c681d78da5215d2f85c17d3c73ff702dd/charts/temporal/templates/web-deployment.yaml#L34-L35). Should that be updated to use internal frontend when that is enabled?

**Comment by robholland** [comment]
No, I don't believe it's intended for the web frontend to use the internal frontend.

Closing this now #602 is merged.
